mathesar/api/db/viewsets/data_files.py

```diff
@@ -44,6 +52,13 @@ def partial_update(self, request, pk=None):
                 status.HTTP_405_METHOD_NOT_ALLOWED
             )
 
+    @extend_schema(
+        request=DataFileSerializer,
+        responses={
+            status.HTTP_201_CREATED: DataFileSerializer,
+            status.HTTP_400_BAD_REQUEST: 'Custom error response for InvalidTableError',
```

Comment:
Shouldn't this be a response body? Also it would be better to wrap it inside `OpenApiResponse`. Look at this [test case](https://github.com/tfranzel/drf-spectacular/blob/1f7cf5d8342f5ed3f619afcf92fb43e3b4ff26cb/tests/test_regressions.py#L1744)

Reply:
I've made the changes, @silentninja could you review it?